modules/core/log/tests/src/Kernel/LogTest.php

```diff
@@ -91,8 +91,8 @@ public function testLogQueryFactory() {
     $log_ids = $this->logQueryFactory->getQuery(['timestamp' => $now])->accessCheck(FALSE)->execute();
     $this->assertNotContains($foo_log->id(), $log_ids, 'Log query results can be filtered by timestamp.');
 
-    // Set the status of one log to complete.
-    $bar_log->set('status', 'complete');
+    // Set the status of one log to "done".
+    $bar_log->set('status', 'done');
```

Comment:
How was this test passing before?

Reply:
Oh, I see... sort of. Shouldn't it fail if you set an invalid status?

Reply:
The reason this didn't cause a test failure is because it's only testing to see if changing this logs status away from `pending` removes it from lists of `pending` logs (which it does).

Technically it's possible to set any value you want as a status. It only gets checked if we run `$entity->validate()` and deal with violations, which we aren't doing in this test.